src/main/java/com/gard/testbed/activitySystem/IBlackboard.java

```diff
@@ -3,7 +3,9 @@
 /**
  * Created by va922kg on 3/15/16.
  */
-public interface IBlackboard {
+public interface IBlackboard extends IObservable{
+    String getId();
```

Comment:
Had them seperate to start with, but wanted to access methods from both in one class (ActivityRunner I think) and if I declared variable as IBlackboard the IObservable was out of scope - found this as one solution on SO. Seperated them again (and cant locate the origional problem that caused me to join them!)